File: JAVA/2351. First Letter to Appear Twice.java
class Solution {
    public char repeatedCharacter(String s) {
        HashSet<Character>x=new HashSet<>();
        for(int i=0;i<s.length();i++){
            x.add(s.charAt(i));
            if(x.size()!=i+1)
            return s.charAt(i);
        }
        return 'a';
    }
}

File: C++/2351. First Letter to Appear Twice.cpp
class Solution {
public:
    char repeatedCharacter(string s) {
        set<char>x;
        for(int i=0;i<s.size();i++){
            x.insert(s[i]);
            if(x.size()!=i+1)
            return s[i];
        }
        return 'a';
    }
};